hibike/hibike_message.py

```diff
@@ -57,9 +57,11 @@
   "GenericError"        : 0xFF
 }
 
+MAX_PACKET_SIZE = 64
```

Comment:
64 bytes is the 'limit' in that back when we were testing arduinos w/ c libraries on the computer, we couldn't get more than 64 bytes (to the arduino) working.

No one tested it now that we're in python.  It may have been a c library problem, and seeing as we've now removed the library, we should test it.